src/app/api/vendors/[id]/check-in/route.ts

```
@@ -0,0 +1,85 @@
+        // 3. Calculate SLA penalty
+        const now = new Date();
+        const slaResult = calculateSLAPenalty(milestone, now);
+
+        // 4. Update milestone with arrival time (trigger will calculate penalty)
+        const { error: updateError } = await supabase
+            .from('vendor_milestones')
+            .update({
+                actual_arrival_timestamp: now.toISOString(),
+                status: 'completed',
+            })
+            .eq('id', milestoneId);
```

ESLint

[error] Replace `············.eq('id'` with `······.eq("id"`

(prettier/prettier)